examples/KelvinHelmholtz/include/simulation_defines/param/radiationConfig.param

```diff
@@ -111,11 +111,6 @@ namespace radiationNyquist
 
 namespace parameters
 {
-/*!enable radiation calculation*/
-#ifndef PARAM_RADIATION
-#define PARAM_RADIATION 0
-#endif
-#define ENABLE_RADIATION PARAM_RADIATION
```

Comment:
Yes correct, ok in that case I will move `PARAM_RADIATION` [directly to the examples](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1931#issuecomment-286720620)